bin/run-ci-tests.bash

```diff
@@ -7,8 +7,21 @@ IFS=$'\n\t'
 # set environment variables
 WCPAY_DIR="$GITHUB_WORKSPACE"
 
+echo 'Updating composer version & Install dependencies...'
 composer self-update 2.0.6 && composer install --no-progress
+
+echo 'Starting MySQL service...'
 sudo systemctl start mysql.service
+
+# On GitHub actions, set MySQL authentication to mysql_native_password instead of caching_sha2_password
+# to prevent DB connection problems with PHP versions less than 7.4
+if [[ -n $CI ]]; then
+	echo "Configuring MySQL to use mysql_native_password"
```

Comment:
For PHP 8, `caching_sha2_password` is the Preferred Authentication Plugin
([ref](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password)), will this lead to issues once we bump up our supported PHP version?

Reply:
@deepakpathania I don't think it would cause an issue because `mysql_native_password` won't be deprecated.